tests/settings.test.ts

```diff
@@ -203,3 +209,95 @@ test.describe('locales', () => {
 		await expect(page.getByText('Servidor')).toBeVisible();
 	});
 });
+
+test('model select supports searching and filtering', async ({ page }) => {
+	await page.goto('/settings');
+	const modelComboBox = page.getByLabel('Available models');
+	await modelComboBox.click();
+
+	await expect(page.getByText('Recently used models', { exact: true })).toBeVisible();
+	await expect(page.getByText('Other models', { exact: true })).toBeVisible();
+
+	await modelComboBox.fill('gemma');
+	await expect(page.getByText('gemma:7b')).toBeVisible();
+	await expect(page.getByText('TheBloke/CodeLlama-70B-Python-AWQ')).not.toBeVisible();
+	await expect(page.getByText('openhermes2.5-mistral:latest')).not.toBeVisible();
+
+	await modelComboBox.fill('');
+	await expect(page.getByText('gemma:7b')).toBeVisible();
+	await expect(page.getByText('TheBloke/CodeLlama-70B-Python-AWQ')).toBeVisible();
+	await expect(page.getByText('openhermes2.5-mistral:latest')).toBeVisible();
+
+	await modelComboBox.fill('nonexistentmodel');
+	await expect(page.getByText('No results')).toBeVisible();
+});
+
+test('models are correctly grouped including recently used models', async ({ page }) => {
+	await page.goto('/settings');
+	await mockTagsResponse(page);
+	await chooseFromCombobox(page, 'Available models', MOCK_API_TAGS_RESPONSE.models[1].name);
+
```

Comment:
There is some duplication of tests with [`ui.test.ts`](https://github.com/fmaclen/hollama/blob/main/tests/ui.test.ts)
I think it makes more sense to move the non-duplicated tests to that file.

You can add a new `test.describe('FieldSelectModel', () => { ... });` section for the new tests.

Also, let's test these features in a the **New session** section, for example: `await page.goto('/sessions/qbhc0q');`, because we are likely to remove the models menu from the **Settings** page after #94